src/Data/Gitbook.elm

```diff
@@ -69,6 +71,12 @@ pathToString path =
         FoodUse ->
             "alimentaire/etapes-du-cycles-de-vie/consommation"
 
+        TextileCircularFootprintFormula ->
+            "textile/cycle-de-vie-des-produits-textiles/circular-footpring-formula-cff"
+
+        TextileCircularFootprintFormula2 ->
+            "textile/cycle-de-vie-des-produits-textiles/etape-1-matieres/calcul-de-limpact-matiere-circular-footprint-formula-cff"
+
```

Comment:
Shouldn't we find another name than adding `2` to the other name?

Reply:
The two pages seem similar, and should probably be merged. In the meantime I've renamed the 2nd one.